src/resources/cdm-detailed.properties

```diff
@@ -101,9 +101,17 @@ spark.cdm.connect.target.password      cassandra
 #                           columns need to be listed.
 #-----------------------------------------------------------------------------------------------------------
 spark.cdm.schema.origin.keyspaceTable                keyspace_name.table_name
+
+# Max TTL value of all non-PK columns will be used for insert on target
```

Comment:
non-PK columns might not be specific enough - not all types support writetime/ttl (collections, counters, etc..). ..so maybe "all ttl-eligible columns" would be a better description..

Reply:
Agree. May be "all ttl-eligible non-PK columns" ?